
----2014-07-15 00:15:52----
Response:{
    "ok": 1
}


----2014-07-15 13:00:58----
(pm25in_1st request)<urlopen error timed out>
Response:{
    "ok": 1
}


----2014-07-15 14:25:16----
'ascii' codec can't encode characters in position 2-3: ordinal not in range(128)

----2014-07-15 14:33:41----
'ascii' codec can't encode characters in position 2-3: ordinal not in range(128)

----2014-07-15 14:45:41----
'ascii' codec can't encode characters in position 2-3: ordinal not in range(128)

----2014-07-15 14:52:25----
'ascii' codec can't encode characters in position 2-3: ordinal not in range(128)

----2014-07-15 14:57:37----

----2014-07-15 14:58:02----

----2014-07-15 14:59:09----

----2014-07-15 15:06:37----

----2014-07-15 15:17:09----

----2014-07-15 15:27:55----

----2014-07-15 15:28:49----
[Errno 9] Bad file descriptor

----2014-07-15 15:30:09----

----2014-07-15 15:42:46----
HTTP Error 400: BAD REQUEST

----2014-07-15 15:51:11----
HTTP Error 400: BAD REQUEST

----2014-07-15 15:55:59----
HTTP Error 400: BAD REQUEST

----2014-07-15 16:04:58----

----2014-07-15 16:09:12----
No JSON object could be decoded

----2014-07-15 16:15:26----
No JSON object could be decoded

----2014-07-15 16:17:32----

----2014-07-15 16:18:06----

----2014-07-15 16:18:41----
__init__() got an unexpected keyword argument 'ensure_ascii'

----2014-07-15 16:18:58----
No JSON object could be decoded

----2014-07-15 16:22:30----
expected string or buffer

----2014-07-15 16:23:01----
No JSON object could be decoded

----2014-07-15 16:24:23----
No JSON object could be decoded

----2014-07-15 16:30:51----

----2014-07-15 16:32:14----
'ascii' codec can't encode characters in position 2-3: ordinal not in range(128)

----2014-07-15 16:37:45----

----2014-07-15 16:40:48----
Response:{
    "ok": 1
}


----2014-07-15 17:05:11----
HTTP Error 500: INTERNAL SERVER ERROR

----2014-07-15 17:18:34----
Response:{
    "ok": 1
}


----2014-07-15 17:23:49----
HTTP Error 400: BAD REQUEST

----2014-07-15 19:30:08----
Response:{
    "ok": 1
}


----2014-07-21 16:44:11----
(pm25in_2nd request)<urlopen error timed out>
Response:{
    "ok": 1
}


----2014-07-21 17:28:50----
'lng'

----2014-07-21 17:29:14----
'lng'

----2014-07-21 17:33:47----
(pm25in_1st request)<urlopen error [Errno 11004] getaddrinfo failed>
<urlopen error [Errno 11004] getaddrinfo failed>

----2014-07-21 17:34:07----
'lng'

----2014-07-21 17:35:22----
'lng'

----2014-07-21 17:35:54----
'lng'

----2014-07-21 17:36:56----
(baidumap api request)<urlopen error timed out>
'NoneType' object is not callable

----2014-07-21 18:06:11----

----2014-07-21 18:11:35----
Response:{
    "ok": 1
}

